content/posts/January-20.md

```diff
@@ -15,3 +15,14 @@ draft: false
 
 - Runs on Fedora 31. ([#303](https://github.com/packit-service/packit-service/pull/303), [#304](https://github.com/packit-service/packit-service/pull/304))
 - Correctly checks list of whitelisted repositories when issue comment is added. ([#309](https://github.com/packit-service/packit-service/pull/309))
+
+## Week 2
+
+### packit
+
+- Incorporates lots of SRPM related improvements. ([#646](https://github.com/packit-service/packit/pull/646), [#648](https://github.com/packit-service/packit/pull/648), [#650](https://github.com/packit-service/packit/pull/650), [#651](https://github.com/packit-service/packit/pull/651), [#652](https://github.com/packit-service/packit/pull/652), [#653](https://github.com/packit-service/packit/pull/653))
```

Comment:
https://github.com/packit-service/packit/pull/648# is about copr

Reply:
uh, I was just lazy, you know :-)
letme change that